src/utils/analysis-utils.ts

```diff
@@ -138,8 +138,10 @@ export const checkBaselineDataLayer = (
   );
 };
 
+export type ApiRequest = {};
```

Comment:
Not sure I get the purpose of these changes?

Reply:
Previously, the fetch method took an ApiData object. ApiData is significantly different than the request I want to make to post an alert, but the fetch method is not specific to ApiData. So I made a type that both ApiData and the new request type inherit from, and changed the fetch method to take that instead.

I'm open to changing this if you think it doesn't make sense.

Reply:
Hmm, as a first thought I would create the request that you want and give the two request type more explicit names and then have the fetch command take a type of "RequestType1 | RequestType2" ?

Reply:
Good suggestion - I updated this in the latest commit